docs/hello_world.md

```diff
@@ -0,0 +1,11 @@
+# Hello World!
+
+After you have learnt how to use the executable go to the directory that you want to make your first program in, create a file with a `.n` extension (preferrably `run.n` but it can be a different name) then add the following code there
+
+```js
+print("Hello World!")
+```
+
+Here `print` is an internal function that will display out any value put into it, here it will output `Hello World!`. Run it using the executable using either `n` if your file is named `run.n` or `n --file [file name]` if your file is named anything else, if you want to check for errors you can add `--check` at the end.
```

Comment:
"internal" implies it's private, that users shouldn't use it, Maybe "built-in" or "native" instead? or "function provided by N"?